.gitlab-ci.yml

```diff
@@ -1,6 +1,8 @@
 include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-github-status-updates.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-commits.yml'
```

Comment:
We need changelogs, we don't release this from `release_tool.` I would align this repo with other internal libraries like `go-lib-micro`.

Reply:
After we reach an agreement, and once you are on it, please add the template also to `go-lib-micro` (it is missing).

Reply:
@oleorhagen your opinions? (you requested change logs :))
I personally kind of like the change logs, but I do not mind keeping only signoffs -- no strong opinions here.

Reply:
we decided that we require only signoffs.